src/game/utils/ParseDirection.js

```diff
@@ -3,37 +3,36 @@ export default function parseDirection(direction) {
     return console.error("Arg must be a string");
   }
 
-  let x = 0,
-    y = 0;
+  let dir = [0, 0];
 
   switch (direction.toUpperCase()) {
```

Comment:
I would move this entire file into Direction.ts and modify it to actually use Directions

Reply:
Yeah this file isn't used, I'm deleting it.